Makefile

```diff
@@ -178,7 +178,7 @@ endif
 	eval $$(opam config env) \
 		&& ${K_BIN}/kompile --debug --main-module ETHEREUM-SIMULATION \
 						    --syntax-module ETHEREUM-SIMULATION .build/ocaml/driver.k --directory .build/llvm \
-						    --backend llvm -ccopt ~/blockchain-k-plugin/plugin-c/crypto.cpp \
+						    --backend llvm -ccopt `pwd`/plugin/plugin-c/crypto.cpp \
```

Comment:
Can we use variable `PLUGIN_SUBMODULE` instead of `pwd/plugin/plugin-c` for consistency?